README.md

```diff
@@ -0,0 +1,38 @@
+# Roulette Telegram Chat
+
+![Man speaking with women](./docs/logo.png)
+The idea is to create a chat for communication between two random people without revealing their identities.
+
+## Use case diagram
+
+![Interaction between user, moderator and system](./docs/useCase.png)
```

Comment:
can we use https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/ for use case diagram?

Reply:
No, we can not:
1. Mermaid [doesn't support](https://mermaid-js.github.io/mermaid/#/sequenceDiagram) use case diagram at all
2. LucidChart provides more enhanced functionality for creating diagrams with WYSIWYG
3. I've already created a diagram

Reply:
ok